CHANGELOG.md

```diff
@@ -105,6 +106,10 @@ outstanding dirty pages that were not flushed. [[GH-2093](https://github.com/has
 * ui: replaces Vault's use of elazarl/go-bindata-assetfs in building the UI with Go's native Embed package [[GH-11208](https://github.com/hashicorp/vault/pull/11208)]
 * ui: updated client tracking config view [[GH-12422](https://github.com/hashicorp/vault/pull/12422)]
 
+DEPRECATIONS:
+
+* auth/kubernetes: deprecate `disable_iss_validation` and `issuer` configuration fields [#127](https://github.com/hashicorp/vault-plugin-auth-kubernetes/pull/127)
```

Comment:
Is there a reason not to include the vault specific PR URL?

Reply:
Good catch! Added in 9f65ec1